.prettierrc

```diff
@@ -1,4 +1,11 @@
 {
+  "semi": true,
   "singleQuote": true,
-  "tabWidth": 2
-}
+  "trailingComma": "none",
+  "tabWidth": 2,
+  "bracketSpacing": true,
+  "arrowParens": "avoid",
+  "importOrder": ["^react(.*)$", "^firebase/(.*)$", "^@(.*)/(.*)$", "^[./]"],
+  "importOrderSeparation": true,
+  "importOrderSortSpecifiers": true
+}
```

Comment:
This are all subjective formatting changes that make this PR way harder to review, as there's thousands of lines of linting changes 😭 

Please make a separate PR for prettier and linting changes.

Reply:
Yeah, sorry about this big change. 

Though linting options and code doesn't match. For example, both .prettierrc and tslint.json states that singleQuotes should be used, but code is using double quotes. I was trying to force the linting config and added some extra configuration. 

I will open another PR with linting changes